config/rbac/role.yaml

```diff
@@ -13,14 +13,16 @@
 # limitations under the License.
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```

Comment:
It would be good to move all the RBAC changes into a separate PR. This will help to make sure everything still works fine with the current permissions and single namespace.

Reply:
Removed RBAC changes.